install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -23,7 +23,7 @@ initContainers:
   - "-x"
   - "{{ annotation .ObjectMeta `traffic.sidecar.istio.io/excludeOutboundIPRanges` .Values.global.proxy.excludeIPRanges }}"
   - "-b"
-  - "{{ annotation .ObjectMeta `traffic.sidecar.istio.io/includeInboundPorts` (includeInboundPorts .Spec.Containers) }}"
+  - "{{ annotation .ObjectMeta `traffic.sidecar.istio.io/includeInboundPorts` `*` }}"
```

Comment:
@lambdai Comment with the link to the backport PR that does the same change in `istio/installer`. If you decided to not change `istio/installer`, explain why in this PR's description.